devdox/tests/unit_test/app/services/test_git_tokens_service.py

```
@@ -19,14 +19,14 @@
     PostGitLabelService,
 )
 from app.utils.auth import UserClaims
-from tests.unit_test.test_doubles.app.repository.get_label_repository_doubles import (
+from models_src.dto.git_label import GitLabelRequestDTO, GitLabelResponseDTO
```

Ruff (0.12.2)

`models_src.dto.git_label.GitLabelRequestDTO` imported but unused

Remove unused import: `models_src.dto.git_label.GitLabelRequestDTO`

(F401)